src/org/labkey/test/tests/SampleTypeLinkedStudyExportTest.java

```diff
@@ -0,0 +1,151 @@
+package org.labkey.test.tests;
+
+import org.jetbrains.annotations.Nullable;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.remoteapi.CommandException;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.Locator;
+import org.labkey.test.TestTimeoutException;
+import org.labkey.test.categories.Daily;
+import org.labkey.test.pages.admin.ExportFolderPage;
+import org.labkey.test.pages.admin.ImportFolderPage;
+import org.labkey.test.params.FieldDefinition;
+import org.labkey.test.params.experiment.SampleTypeDefinition;
+import org.labkey.test.util.DataRegionTable;
+import org.labkey.test.util.PortalHelper;
+import org.labkey.test.util.SampleTypeHelper;
+import org.labkey.test.util.StudyHelper;
+
+import java.io.File;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Arrays;
+import java.util.List;
+
+@Category({Daily.class})
+@BaseWebDriverTest.ClassTimeout(minutes = 2)
+public class SampleTypeLinkedStudyExportTest extends BaseWebDriverTest
+{
+    private final static String SAMPLE_TYPE_PROJECT = "Sample Types Project";
+    private final static String SAMPLE_TYPE = "Samples";
+    private final static String STUDY_EXPORT = "Export Study Test Project";
+    private final static String STUDY_IMPORT = "Import Study Test Project";
```

Comment:
These project names are confusing. The study isn't the thing that gets imported/exported.
```suggestion
    private final static String STUDY_PROJECT = "Target Study Project";
    private final static String SAMPLE_TYPE_IMPORT_PROJECT = "Sample Type Import Project";
```